### PR TITLE
New UNITS of Measure added

### PR DESCRIPTION
**Pull request by vladkorsik**
The PR aimed to add some new Units of Measure that exist either in real data and are supported by not-yet OMOPed Controlled Terminologies ;

### CODE REVIEW

**Comment by vladkorsik** [comment]
@TinyRickC137 Please specifically check the units like '%centimeter watercolumn%'

**Review by AlaikseiKatyshou — COMMENTED** [review]
Reviewed

File: working/manual_changes/2024/2024-07-09.sql [hunk 1]
```diff
@@ -0,0 +1,121 @@
+--add new UCUM concepts
+DO $_$
```
Comment (AlaikseiKatyshou) [inline working/manual_changes/2024/2024-07-09.sql:2]: Why not put all the calls to the AddNewConcept() function within a single DO block?

Comment (vladkorsik) [reply working/manual_changes/2024/2024-07-09.sql:2]: Let's call it common practice/ habits /style ; Separate DO blocks were used for quite a while - so it's a matter of fact. I assume that it's unlikely that changing to a different approach would significantly impact the outcome or its associated cost.

**Review by TinyRickC137 — APPROVED** [review]

**Comment by AlaikseiKatyshou** [comment]
Reviewed

**Review by AlaikseiKatyshou — APPROVED** [review]

**Pull request merged**